main.c

```diff
@@ -408,12 +392,11 @@ int main(void) {
   // Setup Animation Timer
   gptStart(&GPTD_BFTM1, &lightAnimationConfig);
 
-  chThdCreateStatic(waThread1, sizeof(waThread1), NORMALPRIO, Thread1, NULL);
```

Comment:
@Codetector1374 as far as I can tell, the new thread is started with the same priority as the existing one.
I thought this is redundant, but maybe I am not understanding something. If yes, I will remove this last commit.

Reply:
Welcomed change. This was just starter code that I left there and never got around to remove. We don't need the extra thread. This will hopefully help with performance.